packages/lesswrong/components/posts/PostsList2.jsx

```diff
@@ -26,25 +26,36 @@ const PostsList2 = ({ results, loading, count, totalCount, loadMore, networkStat
   //         Alternatively, is there a better way of checking that this is
   //         in fact the best way of checking loading status?
 
-  // TODO-A (2019-2-20): For now, solving this with a flag that determines whether 
+  // TODO-A (2019-2-20): For now, solving this with a flag that determines whether
   //                     to dim the list during loading, so that the pages where that
-  //                     behavior was more important can work fine. Will probably 
+  //                     behavior was more important can work fine. Will probably
   //                     fix this for real when Apollo 2 comes out
-  
+
   const { Loading, PostsItem2, PostsLoadMore, PostsNoResults } = Components
 
   const loadingMore = networkStatus === 2 || networkStatus === 1;
 
   if (!results && loading) return <Loading />
   if ((results && !results.length) && showNoResults) return <PostsNoResults />
 
+  const limit = terms.limit || 10
+  const maybeMorePosts = !!(results && results.length && (results.length >= limit)) // We don't actually know if there are more posts here
```

Comment:
Style nitpick: I'd put this comment on the lines above, rather than on the right.